recipes/phe/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "phe" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: b5785c8e824eb7b02ebbe9b1e2080982d48a10db7b827a6771e04e22da99f473
+
+build:
+  number: 0
+  entry_points:
+    - pheutil = phe.command_line:cli
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+  run_constrained:
```

Comment:
@systemshift  Does anything in `phe` work if `click`, `gmpy2` and `numpy` are not installed?

`run_constrained` is used for controlling the versions of packages so it won't actually do anything here if you don't have a version constraint.

Reply:
@chrisburr  yeah I have read @hmaarrfk's comment regrading this. Unless @wilko77 has something to add, I made pr for him to accept https://github.com/wilko77/staged-recipes/pull/2.

Reply:
In that case the `requires` section under `tests` should also be removed. That section is only normally used for things like `pytest` that really are only needed for testing.

Reply:
I removed the run_constrained section, as it wasn't the right thing.

Our library has a pure python fall-back. However, if there are libraries like gmpy2 available, it will make use of some of the highly optimized functions for better performance.
In the `setup.py` we can specify this in the `extras_require` section. I didn't find an equivalent for the `meta.yaml` file. Thus, I just leave that out.

We do however test the functionality of using those optimized libraries, thus the tests require `click`, `gmpy2` and `numpy` to run.

I made changes to the meta.yaml file and hope that everything is in order now.